src/types/Post.ts

```diff
@@ -0,0 +1,11 @@
+import { Comment } from './Comment';
+import { User } from './User';
+
+export interface Post {
+  userId: number,
+  id: number,
+  title: string,
+  body: string,
+  comments: Comment[] | [],
```

Comment:
You can write just Comment[] as an empty array is also Comment[]

Reply:
Done